### DISCUSSION TITLE
Display Not Refreshing

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#970] [Fixed Issues] [answered]
If I play tracks from a local folder and then select a webradio feed, the metadata and cover art from the local folder remain on the playback page until the screen is manually refreshed. 

I am using replace/play, and confirm replacement is disabled. Library switch to playback option is enabled.

**Comment by rern** [comment, marked as answer]
What is your RPi model?
I've set the options on mine and switching from a local file to Radio Paradise stations updates metadata and cover art properly.
Please try with another file / station.

**Reply by R101G** [reply]
Thanks for the reply @rern. I apologise for wasting your time. A single file or a CD in the album folder was OK. The problem happened when the files added to the playlist were a 2 CD set with each CD in a subfolder. Going back to the library and selecting an RP station after the playlist had finished displayed the RP logo and the first track title from the previous playlist. While I was trying to pin this down, I accidentally invoked a complete rescan of sources, and the problem disappeared. So, apologies again. A previous rescan of changed files did not solve the problem, .

rAudio 1 • 202207101
5.15.53-1 • aarch64
Raspberry Pi 3B Rev 1.2
BCM2837 • 951MiB
4 x Cortex-A53 @ 1.2 GHz
